doc/install.md

```diff
@@ -45,6 +45,8 @@ Contrary to what the documentation in squashfs implies, squashtool and
 libsquash from squash-tools-ng need to be installed globally, as user specific
 path overrides aren't propagated into `make check`'s test envs.
 
+`make check` also requires the golangci-lint to be installed in /bin
```

Comment:
really it's just your $GOPATH/bin here right? https://github.com/project-stacker/stacker/blob/master/Makefile#L41